config.txt

```diff
@@ -63,6 +63,7 @@ dtoverlay=dwc2
 # information across reboots in DRAM
 dtoverlay=ramoops
 
-# Enable the UART (/dev/ttyS0)
+# Enable the UART (/dev/ttyAMA0)
 enable_uart=1
+dtparam=uart0_console
```

Comment:
Can the `dtparam` line be skipped and the `erlinit.config` be changed to use `ttyAMA10`? That seems more in line with using `ttyAMA10` for debugging as the Raspberry Pi docs seem to suggest.

Reply:
Yes, that way we'll have more pins to use freely, so it might be better. I will confirm the `ttyAMA10` output with debug probe, then fix this PR to fit it. Thanks for your review.